hummingbird/ml/operator_converters/sklearn/skl_sv.py

```diff
@@ -18,7 +18,7 @@
 
 class SVC(BaseOperator, torch.nn.Module):
     def __init__(self, kernel, degree, sv, nv, a, b, gamma, coef0, classes, device):
-        super(SVC, self).__init__()
+        super(SVC, self).__init__(classification=True)
```

Comment:
this is due to the container change, right? do we need to change this in other places, or is this sort of a bugfix?

Reply:
I think it needs to go in all classification cases. Hopefully I got all of them. For trees I set the type after the class initialization (because as the trees are implemented we cannot do at instance creation time).